apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
```

Comment:
So we do not check whether a router is being used by another router during deletion? I'm assuming that check will be added in a follow up?